libs/designsystem/calendar/src/calendar.component.ts

```diff
@@ -548,4 +525,76 @@ export class CalendarComponent implements OnInit, AfterViewInit, OnChanges {
     const numberOfYears = endYear - startYear;
     return Array.from({ length: numberOfYears + 1 }, (_, i) => (startYear + i).toString());
   }
+
+  private setFocussedCell(newDate: Date) {
+    const newDay = this.getCell(newDate);
+    if (newDay) {
+      if (this.focussedDay) {
+        this.focussedDay.isFocussed = false;
+      }
+      newDay.isFocussed = true;
+      this.focussedDay = newDay;
+    }
+  }
+
+  private focusDate(newDate: Date | null) {
+    if (!newDate) return;
+    if (this.timezone === 'UTC') {
+      newDate = zonedTimeToUtc(this.subtractTimezoneOffset(newDate), this.timeZoneName);
+    }
+
+    const today = this.todayDate ? startOfDay(this.todayDate) : startOfDay(new Date());
```

Comment:
Consider simplifying:
```suggestion
    const today = startOfDay(this.todayDate ?? new Date());
```